### ISSUE TITLE
[UI] Carousel

### ISSUE THREAD

**Issue by ghost** [Semantic-Org/Semantic-UI#962]
Now a days image slider(carousal) is very common component. I wonder that the team has not yet added a beautiful image slider.

**Comment by atian25** [comment]
+1 for this

**Comment by dragonfire1119** [comment]
+1 also

**Comment by ymind** [comment]
+1 also

**Comment by dickeylth** [comment]
+1 also

**Comment by chumakoff** [comment]
+1

**Comment by jlukic** [comment]
http://www.smashingmagazine.com/2015/02/09/carousel-usage-exploration-on-mobile-e-commerce-websites/

**Comment by leandrotruppa** [comment]
Carousels are not sliders! The carousels is the best way to show which brands you sell or maybe if you have a esports team your sponsors. 

The most simple but nice looking carousel is the one integrated with the default Opencart ecommerce Cms.

**Comment by smilingugh** [comment]
+1

**Comment by cazzoo** [comment]
What about looking arount [OwlCarousel](http://owlgraphic.com/owlcarousel/) that is more than a simple carousel?

**Comment by gabormeszoly** [comment]
What do you think about this?
https://github.com/nolimits4web/Swiper

**Comment by esbanarango** [comment]
+1

**Comment by SunDi3yansyah** [comment]
+1

**Comment by CoreMaker** [comment]
Please refer to this - http://kenwheeler.github.io/slick/.

**Comment by jlukic** [comment]
Slick is really good and would require a lot of work to emulate.

**Comment by CoreMaker** [comment]
Agree. Just wondering if the project has any roadmap to hit?

**Comment by Tumne** [comment]
+ 1

**Comment by ApfelUser** [comment]
+1

**Comment by z0om** [comment]
+1

**Comment by zhunus** [comment]
+, also it will be good to use it on other components like modals, cards...

**Comment by unplugandplay** [comment]
+1

**Comment by behnoodk** [comment]
+1

**Comment by DaniellaGiovanna** [comment]
+1

**Comment by afshinm** [comment]
+1

**Comment by Tsur** [comment]
+1

**Comment by brunoocasali** [comment]
:+1:

**Comment by IonutBajescu** [comment]
+1

**Comment by ronaldo-systemar** [comment]
+1

**Comment by fahim9280** [comment]
+1

**Comment by DanBonehill** [comment]
+1

**Comment by abdelouahabb** [comment]
let's add another +1 around!

**Comment by zhuowenli** [comment]
+1

**Comment by darioshanghai** [comment]
+1

**Comment by jstriedinger** [comment]
+1

**Comment by talha-asad** [comment]
+1

**Comment by destan** [comment]
:+1:

**Comment by jaredjburgess** [comment]
+1

**Comment by serieznyi** [comment]
+1

**Comment by krtschmr** [comment]
+2

**Comment by StanLindsey** [comment]
+1

**Comment by jlukic** [comment]
Carousels man, they never die.

**Comment by chriscubos** [comment]
+1

**Comment by nick-lee** [comment]
+1

**Comment by emzero** [comment]
+1

**Comment by aessig** [comment]
+1

**Comment by wicky-info** [comment]
+1

**Comment by haofu** [comment]
+1

**Comment by KapuzenSohn** [comment]
Guys, please!
Github added a `reaction` Button. There you can add the annoying +1.
When you want to subscribe please just press on the right side the `subscribe` Button.
Thanks!

**Comment by lenovouser** [comment]
Yes, I just wanted to write the same. Would be helpful if you could delete the old `+1` too in my opinion because this issue is really cluttered.

**Comment by Andrianov** [comment]
+1

**Comment by activereset** [comment]
+1

**Comment by 74sharlock** [comment]
+1

**Comment by radiart** [comment]
+1

**Comment by guykatz** [comment]
+1

**Comment by co-entertain** [comment]
+10086

**Comment by nullne** [comment]
+10086

**Comment by lholznagel** [comment]
Guys, please!
When you are intressted in this topic, please click on `Subscribe`.
If you want to to say that this is a nice feature just click on the smiley and press your reaction. Github added that for a reason. One of the reason is that the +1 is annoying for all that subscribed or watch a issue/repo.

Thanks!

**Comment by CodeLuca** [comment]
+1

**Comment by coiby** [comment]
+1

**Comment by coiby** [comment]
+1

**Comment by CodeLuca** [comment]
It's been 2 years, can we do something about this?

**Comment by crewow** [comment]
We need this component as this is the most important for landing page

**Comment by fjbsantiago** [comment]
We could also really use this component for our landing page.

**Comment by pKrav75** [comment]
+1

**Comment by IrinaStancheva** [comment]
+1

**Comment by xiwc** [comment]
+1

**Comment by ramoneariel** [comment]
+

**Comment by fhals** [comment]
+1

**Comment by ivantcholakov** [comment]
http://iridadesign.com/starter-public-edition-4/www/playground/slick/example-2 - An example of Slick-based carousel, quick, without deep integration with Semantic UI.

**Comment by adrianzailic** [comment]
+1

**Comment by rezgui** [comment]
+1

**Comment by rezgui** [comment]
http://glide.jedrzejchalubek.com/

**Comment by andreweugene** [comment]
+1

**Comment by vctrferreira** [comment]
+1

**Comment by vinayakkulkarni** [comment]
Instead of using `+1` to vote, kindly use GitHub reactions to vote.

Cheers! 🍻

**Comment by shajanjp** [comment]
+1

**Comment by guykatz** [comment]
my god, so many people need/want this.
is it so problematic to add it?

**Comment by destan** [comment]
would it help if somebody start a kick-starter or something like that for this issue? it would encourage the maintainer I guess. ideas?

**Comment by krishaamer** [comment]
This would be great!

**Comment by Vextraaa** [comment]
+1

**Comment by dichosov** [comment]
+2

**Comment by rbsl-yasmin** [comment]
Please provide an image slider/ carousel. it's a basic component of any CSS framework.

**Comment by luxlogica** [comment]
Please, **DO NOT ADD A COMMENT WTH  _+1_**.

If you simply wish to be notified of changes to this discussion, press the *'Subscribe'* button at the top of the page.

If you just wish to say that you agree with a previous commenter, you can add your _reaction_ to their comment by clicking on the __*'reaction'* button__ - the one with a plus sign and a smiley face - at the top right corner of the comment itself. 

If someone has already shown a reaction - i.e., if they've already added a 👍 or ❤️ reaction to that comment - then you can add your own reaction simply by clicking the icon itself.

### If you just want to say that you "want a Carousel, too", just click the 👍 icon in the first comment.

**Comment by HipeL** [comment]
Leaving this for reference, I've been using http://fotorama.io/ for some smaller and simplistic stuff since it's pretty easy to implement, so maybe in a future a semantic integration with fotorama could be possible ^^

**Comment by KakitLee** [comment]
I really like using Semantic UI, could you please add carousel?

**Comment by denmasyarikin** [comment]
+1

**Comment by akotulu** [comment]
Fotorama is awesome.

[http://fotorama.io/](http://fotorama.io/)

**Comment by FiringBlanks** [comment]
+1

**Comment by davidalbertonogueira** [comment]
I fully support this request.

**Comment by luxlogica** [comment]
Sad to see this issue marked as 'stale'. It would be helpful to have an official word from the Semantic UI developers as to whether a carousel/slideshow/slider will ever be considered. Despite the no-sayers, carousels and slideshows are an ubiquitous feature of the vast majority of sites. Bootstrap has it. Foundation has it. The new version of UIKit has an amazingly configurable one, allowing us to setup slideshows with ken burns effects and animated captions. 

Having a needed element built-in certainly weighs on the choice of framework to use. The lack of a carousel/slideshow has consistently kept us from using Semantic... :(

**Comment by mtness** [comment]
It's ok for me that sematic-ui doesn't feature a built-in slider mechanic -
since it is a lot of work to incorporate all the different featuresets available today.
Maybe thew page semantic-ui.com/carousel should at least be reactivated to state _why_ there is no slider element available, and a list of recommendations -
for start [slick](http://kenwheeler.github.io/slick/), [swiper](http://idangero.us/swiper/get-started/) and [FlexSlider](https://github.com/woocommerce/FlexSlider) seem to be well-suited solutions for this.

**Comment by hammy2899** [comment]
With the number of contributors who work on SUI and the amount of issues currently I don't think this will be added for some time, also @jlukic has said that he wants to keep the number of components low so the library doesn't become bloated.

Maybe this will be added in a major release sometime in the future!

**Comment by China256** [comment]
+1

**Comment by FiringBlanks** [comment]
@hammy2899 Define bloated

**Comment by crewow** [comment]
still no chance 👍

**Comment by dastan-smailov** [comment]
I belive I can fly

**Comment by codreanulaurentiu** [comment]
The struggle is real. Still waiting for this 5 years later

**Comment by ivantcholakov** [comment]
https://swiperjs.com/

**Comment by Dubkoslove** [comment]
Still waiting
